src/libs/location/src/Character.cpp

```diff
@@ -796,6 +803,29 @@ uint64_t Character::ProcessMessage(MESSAGE &message)
         location->AddDamageMessage(curPos + CVECTOR(0.0f, height, 0.0f), hit, chp, mhp);
     }
         return 1;
+    // Procedural head look
+    case MSG_CHARACTER_HEADLOOK_DEFAULT:
+        curHeadLookState = headlook_default;
+        return 1;
+    case MSG_CHARACTER_HEADLOOK_CAMERA:
+        curHeadLookState = headlook_camera;
+        return 1;
+    case MSG_CHARACTER_HEADLOOK_CHARACTER: {
+        curHeadLookState = headlook_character;
+
+        const entid_t chr = message.EntityID();
+        headLookChrTarget = static_cast<Character *>(EntityManager::GetEntityPointer(chr));
```

Comment:
Don't store ptr - store entid_t instead. Get ptr each time you need it and test against nullptr

Reply:
Thanks for the feedback!
I've fixed it.